test/problems/test_lasso_small.jl

```diff
@@ -194,4 +194,19 @@ using ProximalAlgorithms
 
     end
 
+    @testset "SFISTA" begin
+
+        # SFISTA
+
+        x0 = zeros(T, n)
+        x0_backup = copy(x0)
+        solver = ProximalAlgorithms.SFISTA(tol = 10 * TOL)
+        y, it = solver(x0, f = f2, h = g, Lf = opnorm(A)^2)
```

Comment:
Should we include a test case for a strongly convex problem? Since `mu > 0` is a peculiar feature of this algorithm.

This could either be lasso with strongly convex loss (full column rank data matrix, but requires computing eigenvalues to get a value for `mu`) or a ridge regression problem (in case `mu` is the full convexity modulus)

Reply:
Sure. I'm doing a small lasso example that has positive eigenvalues for `A'A`. It's still a work in progress, but you can look at what I have in the latest push.

Reply:
Done. I finished the strongly convex lasso example.